### DISCUSSION TITLE
Install error

### DISCUSSION THREAD

**Discussion by emageint** [tomatophp/filament-fcm-driver#2] [General]
Root composer.json requires tomatophp/filament-fcm-driver * -> satisfiable by tomatophp/filament-fcm-driver[v1.0.0].
    - tomatophp/filament-fcm-driver v1.0.0 requires tomatophp/filament-alerts ^2.2 -> found tomatophp/filament-alerts[2.2.0, 2.2.1, 2.2.2] but the package is fixed to v2.0.4 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

**Comment by fadymondy** [comment]
that's because v2.0 doesn't support drivers we start driver structure just from v2.2 so to use this driver just update filament-alerts to v2.2
